projects/1.19/assets/ender-zoology/enderzoology/lang/zh_cn.json

```
@@ -0,0 +1,63 @@
+{
+  "block.enderzoology.concussion_charge": "震荡炸药包",
+  "block.enderzoology.confusing_charge": "迷乱炸药包",
+  "block.enderzoology.ender_charge": "末影炸药包",
+  "effect.enderzoology.displacement": "位移",
+  "enchantment.enderzoology.decay": "衰变",
+  "enchantment.enderzoology.decay.desc": "对被攻击者施加凋零效果",
+  "enchantment.enderzoology.repellent": "斥力",
+  "enchantment.enderzoology.repellent.desc": "敌人攻击你时随机传送敌人",
+  "enchantment.enderzoology.soulbound": "灵魂绑定",
+  "enchantment.enderzoology.soulbound.desc": "死亡时保留物品，但魔咒等级可能会随机下降一级",
+  "enchantment.enderzoology.withering": "枯萎",
+  "enchantment.enderzoology.withering.desc": "对被箭击中的敌人施加凋零效果",
+  "entity.enderzoology.concussion_creeper": "震荡苦力怕",
+  "entity.enderzoology.dire_wolf": "冰原狼",
+  "entity.enderzoology.enderminy": "迷你末影人",
+  "entity.enderzoology.fallen_knight": "堕落骑士",
+  "entity.enderzoology.fallen_mount": "堕落战马",
+  "entity.enderzoology.infested_zombie": "被感染的僵尸",
+  "entity.enderzoology.owl": "猫头鹰",
+  "entity.enderzoology.owl_egg": "投掷猫头鹰蛋",

```

```suggestion
  "entity.enderzoology.owl_egg": "猫头鹰蛋",
```
这个改法是遵从原版对鸡蛋实体的命名法，如果觉得不妥就不改了
https://minecraft.fandom.com/zh/wiki/%E9%B8%A1%E8%9B%8B

---

```suggestion
  "entity.enderzoology.owl_egg": "掷出的猫头鹰蛋",
```

---

这个`thrown`再调整一下应该就没有问题了吧